docs/tutorial.md

```diff
@@ -4,7 +4,7 @@ title: Plugin Tutorial
 
 ## Introduction
 
-At the heart of fwupd is a plugin loader that gets run at startup, when devices
+At the heart of fwupd are plugins that gets run at startup, when devices
```

Comment:
Given they're all built in I have to ask is "plugins" still the right name?

It might be, but just food for thought.

Reply:
I wondered too, but I think that they're more plugins than modules or objects. Plus, FwupdPlugin is exported in libfwupd and I'm not super keen on ever bumping that :)